app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,124 @@
+  KongGateway[<img src="https://raw.githubusercontent.com/Kong/docs.konghq.com/main/app/assets/images/icons/kong-gradient.svg" width="15" height="15"/> Kong Gateway]:::resource
+
+  %% Define the Konnect Service Catalog and its subgraphs
+  subgraph KonnectServiceCatalog[Konnect Service Catalog]
+    direction TB
+
+    %% Define the Billing Service subgraph inside Konnect Service Catalog
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService[Kong Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Kong Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
+  KongGateway -- "Kong Gateway services" --> KonnectServiceCatalog
+
+  %% Style the subgraphs
+  class KonnectServiceCatalog konnect
+  class BillingService serviceCatalog
+  class InventoryService serviceCatalog
+{% endmermaid %}
+<!-- vale on-->
+
+> Figure 1: This diagram shows how you can use both external integrations, like GitHub and PagerDuty, as well as built-in integrations like {{site.base_gateway}} to pull resources into Service Catalog. You can then map those resources (like GitHub repositories, PagerDuty services, and {{site.base_gateway}} services) to Service Catalog Services.
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Service Catalog | The canonical system of record for your organization's services in {{site.konnect_short_name}}. This includes all the interactions between them and with external APIs. |
+| Service | A Service is the top-level entity in the Service Catalog. It represents what you might consider a “service”, typically defined as an independent system delivering specific capabilities and owned by a singular team in your organization. It is *not* the same as a {{site.base_gateway}} [service](/gateway/latest/key-concepts/services/). |
+| Integration | These are applications, either internal {{site.konnect_short_name}} applications or external applications, that act as sources from which you can ingest Resources. |
+| Resource | Resources are entities that are injested from Integrations. They can range from infrastructure (for example, a database or a cache) to external services (for example, a code repository, CI/CD infrastructure, an on-call system, documentation, or an API spec). Resources can be mapped to one or more Services. |
+
+## FAQs
```

Comment:
these don't render in the preview.

Reply:
@Guaris Hmmm, they seem to be rendering for me

Reply:
![image](https://github.com/user-attachments/assets/83720e34-1dd4-4b22-88a0-c248aea4a2ed)
This is what they look like for me.

Reply:
^ This is what it looks like for me too! The preview doesn't seem to work for me either.